package.json

```diff
@@ -13,7 +13,7 @@
     "url": "https://github.com/GetTerminus/terminus-ui/issues"
   },
   "scripts": {
-    "//=> Section: Demo App": "==============================",
+    "//////// Section: Demo App": "==============================",
```

Comment:
Help my dumb eyes see the section better. We were getting so many scripts I was experimenting with ways to better separate sections since we can't add comments to pure JSON.

The old format simply didn't jump out enough to me so it wasn't as helpful as it could be. Trying out a new format that jumps out a bit more.